ntp-udp/src/socket.rs

```diff
@@ -179,7 +179,7 @@ impl UdpSocket {
                     .await
                 {
                     Err(_) => {
-                        warn!("Packet without timestamp");
+                        tracing::info!("Packet without timestamp");
```

Comment:
I think this at this point does indicate a problematic condition (we asked the OS for timestamps, but ain't getting them), which the end user should probably want to fix. Hence I think warn is more appropriate

Reply:
Choice: warn, unless we find that this causes real trouble